hooks/pre/30-el7_upgrade_postgresql.rb

```diff
@@ -11,6 +11,7 @@ def postgresql_12_upgrade
 
   execute!(%(scl enable rh-postgresql12 "PGSETUP_INITDB_OPTIONS='--lc-collate=#{collate} --lc-ctype=#{ctype} --locale=#{collate}' postgresql-setup --upgrade"))
   ensure_packages(['postgresql-server'], 'absent')
+  ensure_packages(['postgresql-devel'], 'absent')
   ensure_packages(['postgresql'], 'absent')
```

Comment:
is there a reason those are three calls to `ensure_packages` (and thus 3 Puppet runs)?

Reply:
Initially I wrote some Puppet code to only call Puppet once (and still provide the dependencies) but @ehelms didn't like that so now we're doing it the slow way.